docs/spec/overview.md

```diff
@@ -1,70 +1,88 @@
-
-* _Channels_ define an abstraction between the eventing infrastructure and the
-  consumption and production of the events.
+# Resource Types
 
-* _Feeds_ bridge the event source into the eventing framework.
+The API defines and provides a complete implementation for
+[Subscription](spec.md#kind-subscription), and abstract resource definitions
+for [Channels](spec.md#kind-channel) and [Providers](spec.md#kind-provisioner)
+which may be fulfilled by multiple backing implementations (much like the
+Kubernetes Ingress resource).
 
-* _Flows_ abstract the path of an event from a source takes to reach an event
-  consumer.
+- A **Subscription** describes the transformation of an event and optional
+  forwarding of a returned event.
 
-Typically, _Feeds_ perform out-of-cluster provisioning, while _Channels_ are
-within-cluster management of event delivery. _Flows_ is a higher order wrapper
-used to connect event producers directly with event consumers leveraging 
-_Feeds_ and _Channels_.
+- A **Channel** provides event persistance and fanout of events from a
+  well-known input address to multiple outputs described by _Subscriptions_.
 
-# Resource Types
+<!-- This image is sourced from https://drive.google.com/open?id=10mmXzDb8S_4_ZG_hcBr7s4HPISyBqcqeJLTXLwkilRc -->
 
-The primary resources in the Knative Eventing _Feeds_ API are EventSource,
-EventType and Feed:
+![Resource Types Overview](images/resource-types-overview.svg)
 
-* **EventSource**, an archetype of an event producer.
+- **ClusterProvisioners** implement strategies for realizing backing resources
+  for different implementations of _Channels_ currently active in the eventing
+  system.
 
-* **EventType**, the schema for an event. 
+<!-- This image is sourced from https://drive.google.com/open?id=1o_0Xh5VjwpQ7Px08h_Q4qnaOdMjt4yCEPixRFwJQjh8 -->
 
-* **Feed**, the association between the output of an event producer to the
-  input of an event consumer.
+<img alt="Resource Types ClusterProvisioners" src="images/resource-types-provisioner.svg" width="200">
 
-An example _Feeds_ setup: GitHub would be the _EventSource_, a Pull Request
-notification would be the _EventType_ and the _Feed_ describes the _org/repo_,
-credentials, the specific _EventType_ and the consumer of the _Feed_ events.
+With extendibility and compostability as a goal of Knative Eventing, the
+eventing API defines several resources that can be reduced down to a well
+understood contracts. These eventing resource interfaces may be fulfilled by
+other Kubernetes objects and then composed in the same way as the concreate
+objects. The interfaces are ([Sinkable](interfaces.md#sinkable),
+[Subscribable](interfaces.md#Subscribable),
+[Targetable](interfaces.md#targetable)). For more details, see
+[Interface Contracts](interfaces.md).
 
+## Subscription
 
-The primary resources in the Knative Eventing _Channels_ API are Channel,
-Subscription and Bus:
+**Subscriptions** describe a flow of events from one _Channel_) to the next
```

Comment:
rejected unless you pay me 12 bitcoin.

(this is a far reaching change that is more than this one update, it should be a follow-up because this style is used throughout the document.)

Reply:
Will do.